x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/netty4/SecurityNetty4HttpServerTransportTests.java

```diff
@@ -645,12 +639,11 @@ public void dispatchBadRequest(final RestChannel channel, final ThreadContext th
                 {
                     EmbeddedChannel ch = new EmbeddedChannel(handler);
                     ByteBuf buf = ch.alloc().buffer();
-                    ByteBufUtil.copy(AsciiString.of("GET /url HTTP/1.1"), buf);
-                    buf.writeByte(HttpConstants.LF);
+                    appendAsciiLine("GET /url HTTP/1.1", buf);
                     ByteBufUtil.copy(AsciiString.of("Host: invalid header value"), buf);
                     buf.writeByte(0x01);
-                    buf.writeByte(HttpConstants.LF);
-                    buf.writeByte(HttpConstants.LF);
+                    appendCrLf(buf);
+                    appendCrLf(buf);
```

Comment:
Do we need two CRLFs here? If so, should we have two CRLFs in the other tests too?

Reply:
I was trying to be consistent with what was there. Here I think we're writing a header line that has an invalid character (so we can't use `appendAsciiLine`), and it's the last header so we put multiple newlines in a row.

The other tests will have two newlines by virtue of using `appendAsciiLine`, followed by an explicit `appendCrLf`.